cypress/integration/settings/users/fee&Fine/maximum-number-of-items-borrower-for-loan-type.spec.js

```diff
@@ -0,0 +1,203 @@
+            cy.getLostItemFeesPolicy({ query: `name=="${LOST_ITEM_FEES_POLICY_NAMES.LOST_ITEM_FEES_POLICY}"` });
+            cy.getCirculationRules()
+              .then(rules => {
+                rulesDefaultString = rules.rulesAsText;
+              });
+          })
+        // create circulation rules
+          .then(() => {
+            const requestPolicyId = Cypress.env(CY_ENV.REQUEST_POLICY)[0].id;
+            const noticePolicyId = Cypress.env(CY_ENV.NOTICE_POLICY)[0].id;
+            const overdueFinePolicyId = Cypress.env(CY_ENV.OVERDUE_FINE_POLICY)[0].id;
+            const lostItemFeesPolicyId = Cypress.env(CY_ENV.LOST_ITEM_FEES_POLICY)[0].id;
+            const policy = `l ${loanPolicy.id} r ${requestPolicyId} n ${noticePolicyId} o ${overdueFinePolicyId} i ${lostItemFeesPolicyId}`;
+            const priority = 'priority: number-of-criteria, criterium (t, s, c, b, a, m, g), last-line';
+            const newRule = `${priority}\nfallback-policy: ${policy}\nt ${limitLoanTypeId}: ${policy}`;
+
+            cy.updateCirculationRules({
+              rulesAsText: newRule,
+            });
+          });
+
+        InventoryInstances.createFolioInstanceViaApi({
+          instance: {
+            instanceTypeId: Cypress.env('instanceTypes')[0].id,
+            title: instanceTitle,
+          },
+          holdings: [{
+            holdingsTypeId: Cypress.env('holdingsTypes')[0].id,
+            permanentLocationId: Cypress.env('locations')[0].id,
+          }],
+          items: testItems
+        })
+          .then(specialInstanceIds => {
+            testInstanceIds = specialInstanceIds;
+          });
+      });
+
+    cy.createTempUser([
+      permissions.checkoutCirculatingItems.gui
+    ])
+      .then(userProperties => {
+        user = userProperties;
+        servicePoint = NewServicePoint.getDefaulServicePoint();
+        ServicePoints.createViaApi(servicePoint.body);
+        cy.addServicePointToUser([servicePoint.body.id],
+          user.userId, servicePoint.body.id);
+      })
+      .then(() => {
+        cy.login(user.username, user.password);
+      });
+  });
+
+  after(() => {
+    limitTestItems.forEach(item => {
+      CheckInActions.createItemCheckinApi({
+        itemBarcode: item.barcode,
+        servicePointId: servicePoint.body.id,
+        checkInDate: '2021-09-30T16:14:50.444Z',
+      });
+    });
+    cy.wrap(limitTstInstanceIds.holdingIds.forEach(holdingsId => {
+      cy.wrap(holdingsId.itemIds.forEach(itemId => {
+        cy.deleteItem(itemId);
+      })).then(() => {
+        cy.deleteHoldingRecord(holdingsId.id);
+      });
+    })).then(() => {
+      cy.deleteInstanceApi(limitTstInstanceIds.instanceId);
+    });
+    testItems.forEach(item => {
+      CheckInActions.createItemCheckinApi({
+        itemBarcode: item.barcode,
+        servicePointId: servicePoint.body.id,
+        checkInDate: '2021-09-30T16:14:50.444Z',
```

Comment:
why this date is used? Try to use current date minus half of year, for example

Reply:
I changed it